### ISSUE TITLE
Remove Google Translate from footer

### ISSUE THREAD

**Issue by magjarvis** [ATLAS-Illinois/Illinois-Framework-Theme#1313]
Several accessibility reviews have flagged the Google Translate dropdown as not having focus, making it inaccessible to screen reader users. 

The Google Translate dropdown will be removed in the 6.0 release of the Illinois Drupal Framework.

There is a config setting that will need to be removed from the theme in addition to removing it from the twig files. The files that should be modified are in the pull request at https://github.com/web-illinois/illinois_framework_theme/pull/1068/changes. It would be worthwhile to do a search across the theme files to make sure there aren't other spots it might be included.
